pikelet/src/surface/lex.rs

```diff
@@ -0,0 +1,190 @@
+use logos::Logos;
+use std::fmt::{self, Display, Formatter};
```

Comment:
I generally just import `fmt` and refer to `Display` and `Formatter` through that path.